packages/ds-healthcare-gov/src/components/Footer/InlineLinkLists.tsx

```diff
@@ -59,25 +55,27 @@ const InlineLinkLists = function (props: InlineLinkListsProps) {
 
   return (
     <div className="ds-l-container">
-      <ul className="ds-c-list ds-c-list--bare ds-u-font-size--small ds-u-margin-bottom--1">
+      <ul role="list" className="hc-c-footer__list ds-u-margin-bottom--1">
```

Comment:
since lists aren't styled, the `role="list"` was added to all ul tags

Reply:
removed some repetitive util classes and moved into CSS file under `hc-c-footer` to reduce lines of code